packages/samples/sudt/src/actors/sudt.model.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * @module src/actors/omnilock.model
+ * @description
+ * This is the actor model for omnilock, which is used to gather omnilock cells to generate record models.
+ */
+
+import {
+  ActorProvider,
+  Omnilock,
+  Param,
+  ActorReference,
+  CellPattern,
+  JSONStore,
+  OutPointString,
+  SchemaPattern,
+  UpdateStorageValue,
+  LockFilter,
+  TypeFilter,
+  Sudt,
+} from '@ckb-js/kuai-models';
+import type { Cell, HexString, Script } from '@ckb-lumos/base';
+import { number, bytes } from '@ckb-lumos/codec';
+import { InternalServerError } from 'http-errors';
+import { BI, utils, config } from '@ckb-lumos/lumos';
+import { TX_FEE } from '../const';
+
+/**
+ * add business logic in an actor
+ */
+@ActorProvider({ ref: { name: 'sudt', path: `/:lockArgs/:typeArgs/` } })
+@LockFilter()
+@TypeFilter()
+@Omnilock()
+@Sudt()
+export class SudtModel extends JSONStore<Record<string, never>> {
+  constructor(
+    @Param('lockArgs') lockArgs: string,
+    @Param('typeArgs') typeArgs: string,
+    _schemaOption?: void,
+    params?: {
+      states?: Record<OutPointString, never>;
+      chainData?: Record<OutPointString, UpdateStorageValue>;
+      cellPattern?: CellPattern;
+      schemaPattern?: SchemaPattern;
+    },
+  ) {
+    super(undefined, { ...params, ref: ActorReference.newWithFilter(SudtModel, `/${lockArgs}/${typeArgs}/`) });
+    if (!this.lockScript || !this.typeScript) {
+      throw new Error('lock script and type script are required');
+    }
+    this.registerResourceBinding();
+  }
+
+  get meta(): Record<'capacity' | 'sudtBalance', string> {
+    let capacity = BigInt(0);
+    let sudtBalance = BigInt(0);
+    Object.values(this.chainData).forEach((v) => {
+      capacity += BigInt(v.cell.cellOutput.capacity ?? 0);
+      sudtBalance += number.Uint128LE.unpack(v.cell.data.slice(0, 34)).toBigInt();
+    });
+    return {
+      capacity: capacity.toString(),
+      sudtBalance: sudtBalance.toString(),
+    };
+  }
+
+  send(lockScript: Script, amount: HexString) {
+    const CONFIG = config.getConfig();
+    const sudtCell: Cell = {
+      cellOutput: {
+        capacity: BI.from('14400000000').toHexString(),
```

Comment:
`14400000000` is a magic number and why the capacity of recipient is set as a fixed value